.gitlab-ci.yml

```diff
@@ -9,7 +9,7 @@ thud-wandboard-mesa-weston-wpe-2.20:
     - pushd sources/meta-webkit
     - git checkout $CI_COMMIT_SHA
     - popd
-    - source setup-environment wandboard-mesa-wpe-2.20 --update-config
+    - source setup-environment wandboard-mesa-wpe-2.20 wandboard-mesa browsers wandboard mesa-wpe-220 --update-config
```

Comment:
Version numbers without separator can quickly become very confusing. Can you use a separator? - or _ ?